pkgs/development/libraries/librsvg/default.nix

```diff
@@ -31,11 +33,17 @@ stdenv.mkDerivation rec {
     sha256 = "//thsIzVKCqq4UegKzBRZqdCb60iqLlCdwjw8vxCbrw=";
   };
 
+  cargoVendorDir = "vendor";
+
+  strictDeps = true;
```

Comment:
I think it makes sense to have it just before the build inputs, because it changes how they work (somewhat), so it's important to have seen that just before seeing the build inputs.